### ISSUE TITLE
Support adding custom response header

### ISSUE THREAD

**Issue by xqliang** [fabiolb/fabio#370]
My Fabio is configured receive all incoming traffic from an existing gateway(Nginx), and I use gateway(Nginx) to write access log.

~~~
                               +- HTTP -> fabio -+-> service-a (host-a)
                               |                 |
internet -- HTTP/HTTPS --> LB -+- HTTP -> fabio -+-> service-b (host-b)
                               |                 |
                               +- HTTP -> fabio -+-> service-c (host-c)
~~~

I want to log service name in the Nginx access log, can Fabio support adding custom response header(e.x X-Fabio-Service-Name)?

**Comment by magiconair** [comment]
No, but you use fabio to write the access log in the same format and there you can add the access log. 

https://github.com/fabiolb/fabio/blob/master/fabio.properties#L419-L467

If that doesn't solve your problem then I can think about how to generalize that since you probably want to configure this either globally or per route. I'd need to think about that a bit.

**Comment by xqliang** [comment]
Thank you, I need to use Nginx to write access log for monitoring the entire request time consuming (including Nginx to Fabio). 

Current, global configuration will meet my needs.
